docs/source/persona_pages/rookie_teams/rookie_teams.rst

```diff
@@ -116,5 +116,5 @@ resource you want to explore!
    :titlesonly:
    :hidden:
 
-   ../../stubs/faqs
+   /faq/faqs
```

Comment:
Do not "hang" this off the rookie team persona, please.

Let's create a new `.. toctree::` on the main toc (in the main index.rst file).
-  Let's caption it with "Team Resources"
-  Let's hang this new faq onto it
-  Let's put it between "Additional Tools" and "Team Freebies and Discounts" (so that it's the next-to-last item in the toctree).